courses/free-courses-en.md

```diff
@@ -26,6 +26,7 @@
 * [Deep Learning](#deep-learning)
 * [DevOps](#devops)
 * [Docker](#docker)
+* [Elastic Stack](#elastic-stack)
```

Comment:
Our lists use "Elastic" and "Elasticsearch" as categories, and the books-subjects list puts Elastic search under "search Engines" we also have a Kibana resource under a generic heading. Maybe easier to add the elastic stack resources in a separate PR that organizes Elastic resources coherently. Also, do you think Kibana and ElasticSearch really belong in the same category? or in an Elastic.co category off to themselves?